test/itemlist_formaction.cpp

```diff
@@ -536,8 +503,71 @@ TEST_CASE("OP_EDITFLAGS arguments are added to an item's flags", "[itemlist_form
 
 }
 
-TEST_CASE("process_op(OP_SAVE)", "[itemlist_formaction]") {
-	//NOTIMPL
+TEST_CASE("OP_SAVE writes an article's attributes to the specified file", "[itemlist_formaction]") {
+	controller c;
+	newsbeuter::view v(&c);
```

Comment:
Funny it's only now I notice this: why is `view` here qualified? We have `using namespace newsbeuter` at the beginning of the file, `view v(&c)` should work just fine.